packages/wallet/test/services/util/persistentTrackerSubjects.test.ts

```diff
@@ -44,7 +44,7 @@ describe('PersistentCollectionTrackerSubject', () => {
       const tracker$ = new PersistentCollectionTrackerSubject<string>(() => source$, store);
       expectObservable(tracker$.asObservable()).toBe('a-b--c', { a, b, c });
     });
-    expect(store.setAll).toBeCalledTimes(2);
+    expect(store.setAll).toBeCalledTimes(4);
```

Comment:
This is because `next` now also calls `store.setAll`

Reply:
Suggestion in [this comment](https://github.com/input-output-hk/cardano-js-sdk/pull/228#discussion_r854838987) might help